src/App.jsx

```diff
@@ -14,33 +15,86 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const ABC = '1';
+const L = '2';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function Sort(arr, reverse, sort) {
```

Comment:
The function name `Sort` should start with a lowercase letter to follow JavaScript naming conventions for functions. Consider renaming it to `sort`.